connectors/cli.py

```diff
@@ -6,7 +6,15 @@
 """
 Command Line Interface.
 
-Parses arguments and call run() with them.
+This is the main entry point of the framework. When the project is installed as
+a Python package, an `elastic-ingest` executable is added in the PATH.
+
+
+`main` is doing the following:
+
+- parses the arguments
+- reads the config file
+- starts the event loop against a series of services
```

Comment:
Is this needed since the `main` function is already documented?

Reply:
The intent of the module docstring is to offer a short explanation of what this module does without having to look around inside the module. Ideally, it should not be duplicating too much content -- so yeah we could reduce this for sure.